uo_sys_framework/FiremissionModule/cfgFunctions/common/fn_RegisterForwardObserver.sqf

```diff
@@ -1,67 +1,67 @@
 #include "..\..\Global\defs.hpp"
 if (isServer) then {
-	_handle = _this spawn {
-		private _observer = _this select 0;
-		private _batteries = _this select 1;
-		private _minimumKnowledge = _this select 2;
-		private _minRange = _this select 3;
-		private _range = _this select 4;
-		private _standardDispersion = _this select 5;
-		private _standardRoundCount = _this select 6;
-		private _standardRoundSalvo = _this select 7;
-		private _standardRoundSalvoWait = _this select 8;
-		private _minSpottedDistance = _this select 9;
-		private _standardRound = _this select 10;
-		private _obsSide = side _observer;
+    _handle = _this spawn {
```

Comment:
Use of params instead of private and _this select. THis applies to most functions in this module.